src/lib/dns-server/service_widget_helpers.rb

```diff
@@ -0,0 +1,22 @@
+require "cwm/service_widget"
+require "yast2/system_service"
+
+module Y2DnsServer
+  module ServiceWidgetHelpers
+    # Returns the 'named' system service
+    #
+    # @return [Yast2::SystemService] 'named' system service
+    def service
+      @service ||= Yast2::SystemService.find("named")
+    end
+
+    # Returns the status widget for service
+    #
+    # @return [::CWM::ServiceWidget] service status widget
```

Comment:
NP: is it necessary to explicitly add `::` for top-level name resolution? In case there is no conflict, I would not add it.

Reply:
I had problems avoiding them, but I can't remember if was here. However, after that I started to use `::CWM::ServiceWidget` in all modules, for consistence. Shall I to use `::` only when needed?